### DISCUSSION TITLE
Can you have a variable for Take Profit?

### DISCUSSION THREAD

**Discussion by ScuzzyHoulihan** [hootnot/oanda-api-v20#189] [Q&A] [answered]
//observe cmd blow. Can I put a variable as the take profit? 
execute_order2(instrument=instrument.get('WTICO_USD'), units=-1, STOP_LOSS=120, TAKE_PROFIT=80)

In particular I am looking to compare two positions to = "TAKE_PROFIT'. If one is negative and  the other positive but combined say  = 50 pips profit close both positions as a take profit.

Hope that makes sense.

**Comment by hootnot** [comment, marked as answer]
You can code whatever you like. So, variables is possible. Two positions in the same instrument: you can ask for the position information instead of tradeinfo and close the position. Closing a position closes all trades related to that position
